apps/storybook/src/Customization.mdx

```diff
@@ -146,7 +146,7 @@ as you see fit. For instance, if you'd like to change the color of the curve of
 | `--h5w-selector-option-focus--outlineColor` | `gray`                                       | Outline color of focused option      |
 | `--h5w-selector-groupLabel--color`          | `gray`                                       | Text color of group labels in menu   |
 
-##### Domain slider
+##### Domain widget
 
 | Name                                                 | Default                    | Description                                    |
```

Comment:
Do we need to change the CSS variable names ?

Reply:
I'll do it after I rename/create the other components, because most will not likely change (if `ScaledSlider` becomes `DomainSlider` and is exported, then the variables can still be called `--h5w-domainSlider-<...>`.

Reply:
Sure. Just wanted to be sure that you had it in mind but it was an unneeded concern apparently :wink: